2022-01-10 12:36:37,916	INFO head.py:128 -- Dashboard head grpc address: 0.0.0.0:34451
2022-01-10 12:36:37,916	INFO dashboard.py:92 -- Setup static dir for dashboard: /home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/client/build
2022-01-10 12:36:37,917	INFO head.py:55 -- Connect to GCS at 131.215.21.210:46129
2022-01-10 12:36:37,920	INFO utils.py:200 -- Get all modules by type: DashboardHeadModule
2022-01-10 12:36:38,135	WARNING tune_head.py:22 -- tune module is not available: cannot import name 'Analysis' from 'ray.tune' (/home/reza/anaconda3/lib/python3.8/site-packages/ray/tune/__init__.py)
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.actor.actor_head.ActorHead'>
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.event.event_head.EventHead'>
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.job.job_head.JobHead'>
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.log.log_head.LogHead'>
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.node.node_head.NodeHead'>
2022-01-10 12:36:38,136	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.reporter.reporter_head.ReportHead'>
2022-01-10 12:36:38,137	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.snapshot.snapshot_head.APIHead'>
2022-01-10 12:36:38,137	INFO head.py:168 -- Loading DashboardHeadModule: <class 'ray.dashboard.modules.tune.tune_head.TuneController'>
2022-01-10 12:36:38,137	INFO head.py:173 -- Loaded 8 modules.
2022-01-10 12:36:38,139	INFO head.py:253 -- Dashboard head http address: 127.0.0.1:8265
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /> -> <function Dashboard.get_index at 0x7f3294ffddc0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /favicon.ico> -> <function Dashboard.get_favicon at 0x7f3294ffdee0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <StaticResource  /static -> PosixPath('/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/client/build/static')> -> <bound method StaticResource._handle of <StaticResource  /static -> PosixPath('/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/client/build/static')>>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /logical/actor_groups> -> <function ActorHead.get_actor_groups at 0x7f3294715820>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /logical/actors> -> <function ActorHead.get_all_actors[cache ttl=2, max_size=128] at 0x7f3294715940>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /logical/kill_actor> -> <function ActorHead.kill_actor at 0x7f3294715af0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /events> -> <function EventHead.get_event[cache ttl=2, max_size=128] at 0x7f32947211f0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/version> -> <function JobHead.get_version at 0x7f3294559c10>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <DynamicResource  /api/packages/{protocol}/{package_name}> -> <function JobHead.get_package at 0x7f3294559dc0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [PUT] <DynamicResource  /api/packages/{protocol}/{package_name}> -> <function JobHead.upload_package at 0x7f3294559f70>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [POST] <PlainResource  /api/jobs/> -> <function JobHead.submit_job at 0x7f329455a160>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [POST] <DynamicResource  /api/jobs/{job_id}/stop> -> <function JobHead.stop_job at 0x7f329455a310>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <DynamicResource  /api/jobs/{job_id}> -> <function JobHead.get_job_status at 0x7f329455a4c0>
2022-01-10 12:36:38,141	INFO head.py:271 -- <ResourceRoute [GET] <DynamicResource  /api/jobs/{job_id}/logs> -> <function JobHead.get_job_logs at 0x7f329455a670>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /log_index> -> <function LogHead.get_log_index at 0x7f32945b50d0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /log_proxy> -> <function LogHead.get_log_from_proxy at 0x7f32945b51f0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /nodes> -> <function NodeHead.get_all_nodes[cache ttl=2, max_size=128] at 0x7f32945b5c10>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <DynamicResource  /nodes/{node_id}> -> <function NodeHead.get_node[cache ttl=2, max_size=128] at 0x7f32945b5dc0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /memory/memory_table> -> <function NodeHead.get_memory_table at 0x7f32945b5f70>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /memory/set_fetch> -> <function NodeHead.set_fetch_memory_info at 0x7f32945680d0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /node_logs> -> <function NodeHead.get_logs at 0x7f32945681f0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /node_errors> -> <function NodeHead.get_errors at 0x7f3294568310>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/launch_profiling> -> <function ReportHead.launch_profiling at 0x7f329445f670>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/ray_config> -> <function ReportHead.get_ray_config at 0x7f329445f790>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/cluster_status> -> <function ReportHead.get_cluster_status at 0x7f329445f8b0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/actors/kill> -> <function APIHead.kill_actor_gcs at 0x7f32943e9e50>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /api/snapshot> -> <function APIHead.snapshot at 0x7f32943e9f70>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /tune/info> -> <function TuneController.tune_info at 0x7f3259f889d0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /tune/availability> -> <function TuneController.get_availability at 0x7f3259f88af0>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /tune/set_experiment> -> <function TuneController.set_tune_experiment at 0x7f3259f88c10>
2022-01-10 12:36:38,142	INFO head.py:271 -- <ResourceRoute [GET] <PlainResource  /tune/enable_tensorboard> -> <function TuneController.enable_tensorboard at 0x7f3259f88d30>
2022-01-10 12:36:38,143	INFO head.py:271 -- <ResourceRoute [GET] <StaticResource  /logs -> PosixPath('/home/reza/Dropbox/Prog/MERA/temp/session_2022-01-10_12-36-36_022669_18798/logs')> -> <bound method StaticResource._handle of <StaticResource  /logs -> PosixPath('/home/reza/Dropbox/Prog/MERA/temp/session_2022-01-10_12-36-36_022669_18798/logs')>>
2022-01-10 12:36:38,143	INFO head.py:272 -- Registered 32 routes.
2022-01-10 12:36:38,143	INFO datacenter.py:69 -- Purge data.
2022-01-10 12:36:38,144	INFO event_utils.py:123 -- Monitor events logs modified after 1641845197.9255848 on /home/reza/Dropbox/Prog/MERA/temp/session_2022-01-10_12-36-36_022669_18798/logs/events, the source types are ['GCS'].
2022-01-10 12:36:38,147	INFO reporter_head.py:138 -- Subscribed to RAY_REPORTER:*
2022-01-10 12:36:38,147	INFO actor_head.py:71 -- Subscribed to ACTOR:*
2022-01-10 12:36:38,147	INFO actor_head.py:81 -- Getting all actor info from GCS.
2022-01-10 12:36:38,148	INFO node_head.py:249 -- Subscribed to <_Sender name:b'RAY_LOG_CHANNEL', is_pattern:False, receiver:<Receiver is_active:True, senders:1, qsize:0>>
2022-01-10 12:36:38,148	INFO node_head.py:309 -- Subscribed to b'ERROR_INFO:*'
2022-01-10 12:36:38,148	INFO actor_head.py:107 -- Received 0 actor info from GCS.
2022-01-10 12:36:38,466	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:36:42,693	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:36:47,626	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:36:52,661	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:36:57,695	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:02,729	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:07,665	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:12,702	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:17,736	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:22,671	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:27,707	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:32,741	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:37,675	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:42,710	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:47,744	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:52,778	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:37:57,711	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:02,746	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:07,780	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:12,714	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:17,748	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:22,786	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:27,823	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:32,757	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:37,793	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:42,828	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:47,760	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:52,795	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:38:57,828	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:39:02,861	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:39:07,795	INFO node_head.py:270 -- Received a log for 131.215.21.210 and autoscaler
2022-01-10 12:39:12,700	ERROR node_head.py:241 -- Error updating node stats of eabe26399494717f195118c1a83a601da65454449c98ab23fd7c587d.
Traceback (most recent call last):
  File "/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/modules/node/node_head.py", line 234, in _update_node_stats
    reply = await stub.GetNodeStats(
  File "/home/reza/.local/lib/python3.8/site-packages/grpc/aio/_call.py", line 285, in __await__
    raise _create_rpc_error(self._cython_call._initial_metadata,
grpc.aio._call.AioRpcError: <AioRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses"
	debug_error_string = "{"created":"@1641847152.700407905","description":"Failed to pick subchannel","file":"src/core/ext/filters/client_channel/client_channel.cc","file_line":4133,"referenced_errors":[{"created":"@1641847152.700405945","description":"failed to connect to all addresses","file":"src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc","file_line":397,"grpc_status":14}]}"
>
2022-01-10 12:39:13,209	ERROR node_head.py:133 -- Error updating nodes.
Traceback (most recent call last):
  File "/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/modules/node/node_head.py", line 99, in _update_nodes
    nodes = await self._get_nodes()
  File "/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/modules/node/node_head.py", line 83, in _get_nodes
    reply = await self._gcs_node_info_stub.GetAllNodeInfo(
  File "/home/reza/.local/lib/python3.8/site-packages/grpc/aio/_call.py", line 285, in __await__
    raise _create_rpc_error(self._cython_call._initial_metadata,
grpc.aio._call.AioRpcError: <AioRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses"
	debug_error_string = "{"created":"@1641847153.208872416","description":"Failed to pick subchannel","file":"src/core/ext/filters/client_channel/client_channel.cc","file_line":4133,"referenced_errors":[{"created":"@1641847153.208870391","description":"failed to connect to all addresses","file":"src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc","file_line":397,"grpc_status":14}]}"
>
2022-01-10 12:39:13,209	ERROR head.py:90 -- Got RpcError when checking GCS is alive
Traceback (most recent call last):
  File "/home/reza/anaconda3/lib/python3.8/site-packages/ray/dashboard/head.py", line 83, in _check_once_synchrounously
    reply = self.gcs_heartbeat_info_stub.CheckAlive(
  File "/home/reza/.local/lib/python3.8/site-packages/grpc/_channel.py", line 826, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/reza/.local/lib/python3.8/site-packages/grpc/_channel.py", line 729, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses"
	debug_error_string = "{"created":"@1641847153.209295906","description":"Failed to pick subchannel","file":"src/core/ext/filters/client_channel/client_channel.cc","file_line":4133,"referenced_errors":[{"created":"@1641847153.209293953","description":"failed to connect to all addresses","file":"src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc","file_line":397,"grpc_status":14}]}"
>
